check/test.go

```diff
@@ -176,7 +200,17 @@ func (t testItem) execute(s string) *testOutput {
 func (t testItem) evaluate(s string) *testOutput {
 	result := &testOutput{}
 
-	match, value, err := t.findValue(s)
+	match, value, err := t.findFlag(s)
+	if err != nil || !match {
```

Comment:
I think we can avoid changes (or at least reduce) changes to this function by having `t.findValue(s)` work for all three types (currently it does two, so let's add Env into it if possible). 

So I think this code wouldn't need to change, but just in case it does - if this returns there's an error, let's handle it. That's a different case from not finding a match.

Reply:
✔️